core/dbt/events/functions.py

```diff
@@ -188,7 +171,9 @@ def create_debug_text_log_line(e: T_Event) -> str:
     color_tag: str = reset_color()
     ts: str = get_ts().strftime("%H:%M:%S.%f")
     scrubbed_msg: str = scrub_secrets(e.message(), env_secrets())
-    level: str = e.level_tag() if len(e.level_tag()) == 5 else f"{e.level_tag()} "
+    # What's the point of this?
+    # level: str = e.level_tag() if len(e.level_tag()) == 5 else f"{e.level_tag()} "
```

Comment:
This is literally to make the text output line up.  The `else` has an extra space in it.  See https://github.com/dbt-labs/dbt-core/pull/4303.  Probably should have a comment as to what it's doing.

Reply:
Ah. Yeah, I did miss the extra space.